packages/opal-server/opal_server/scopes/api.py

```diff
@@ -96,4 +101,14 @@ async def get_scope_policy(
             bundle = fetcher.make_bundle(base_hash)
             return bundle
 
+    @router.post("/{scope_id}/policy_update")
+    async def notify_new_policy(
+        *,
+        scope_id: str = Path(..., description="Scope ID"),
+        notification: PolicyUpdateMessageNotification,
+    ):
+        async with ScopedServerSideTopicPublisher(pubsub, scope_id) as publisher:
+            publisher.publish(notification.topics, notification.update)
+            await publisher.wait()
```

Comment:
you probably want to return a response with status code 204.
not sure fastapi does it for you and it can cause weird issues.
check that it works correctly.